File: ds_&_algorithms/algorithm_analysis/polynomial_time_complexity/java/PolTimeComp.java
package algorithm_analysis.polynomial_time_complexity.java;

public class PolTimeComp {
    public void printFunc(int n) {
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= n; j++) {
                System.out.println("i = " + i + ", j = " + j);
            }
            System.out.println("End of inner loop");
        }
        System.out.println("End of outer loop");
    }

    public static void main(String[] args) {
        int value = 3;
        PolTimeComp obj = new PolTimeComp(); // Creating an instance of the class
        System.out.println("The value of the range is as follows:");
        obj.printFunc(value); // Correct method call
    }
}

File: ds_&_algorithms/algorithm_analysis/polynomial_time_complexity/go/PolTimeComp.go
package main

import "fmt"

func polTimeComp(n int) {
	for i := 1; i <= n; i++ {
		for j := 1; j <= n; j++ {
			fmt.Printf("i=%d, j=%d", i, j)
		}
		fmt.Println("End of inner loop")
	}
	fmt.Println("End of outer loop")
}

func main() {
	var value int = 3

	fmt.Println("The value of the loop is:\n")

	polTimeComp(value)
}
